tests/frontDesk/visits.js

```diff
@@ -97,33 +97,33 @@ step("Verify vitals", async function () {
     var diagnosisFile = gauge.dataStore.scenarioStore.get("diagnosisFile")
     var medicalDiagnosis = JSON.parse(fileExtension.parseContent(diagnosisFile))
-    assert.ok(await text(medicalDiagnosis.diagnosis.diagnosisName, toLeftOf(medicalDiagnosis.diagnosis.certainty, toLeftOf(medicalDiagnosis.diagnosis.order)), within($("//section[@id='Diagnosis']"))).exists())
+    assert.ok(await text(medicalDiagnosis.diagnosis.diagnosisName, toLeftOf(medicalDiagnosis.diagnosis.certainty, toLeftOf(medicalDiagnosis.diagnosis.order)), within($("//section[@id='Diagnoses']"))).exists())
 });
 
 step("Verify history & examination in patient clinical dashboard", async function () {
     var historyAndExaminationDetails = gauge.dataStore.scenarioStore.get("historyAndExaminationDetails")
-    assert.ok(await text(`${historyAndExaminationDetails.Chief_Complaints[0].Chief_Complaint} since ${historyAndExaminationDetails.Chief_Complaints[0].for} ${historyAndExaminationDetails.Chief_Complaints[0].for_frequency}`, toRightOf("Chief Complaint"), within($("#History-and-Examinations"))).exists())
-    assert.ok(await text(`${historyAndExaminationDetails.Chief_complaint_notes}`, within($("#History-and-Examinations")), toRightOf("Chief Complaint Notes")).exists())
-    assert.ok(await text(`${historyAndExaminationDetails.History_Notes}`, within($("#History-and-Examinations")), toRightOf("History Notes")).exists())
-    assert.ok(await text(`${historyAndExaminationDetails.Examination_notes}`, within($("#History-and-Examinations")), toRightOf("Examination Notes")).exists())
-    assert.ok(await text(`${historyAndExaminationDetails.Smoking_History}`, within($("#History-and-Examinations")), toRightOf("Smoking History")).exists())
+    assert.ok(await text(`${historyAndExaminationDetails.Chief_Complaints[0].Chief_Complaint} since ${historyAndExaminationDetails.Chief_Complaints[0].for} ${historyAndExaminationDetails.Chief_Complaints[0].for_frequency}`, toRightOf("Chief Complaint"), within($(" #History-and-Examinations "))).exists())
+    assert.ok(await text(`${historyAndExaminationDetails.Chief_complaint_notes}`, within($(" #History-and-Examinations ")), toRightOf("Chief Complaint Notes")).exists())
+    assert.ok(await text(`${historyAndExaminationDetails.History_Notes}`, within($(" #History-and-Examinations ")), toRightOf("History Notes")).exists())
+    assert.ok(await text(`${historyAndExaminationDetails.Examination_notes}`, within($(" #History-and-Examinations ")), toRightOf("Examination Notes")).exists())
+    assert.ok(await text(`${historyAndExaminationDetails.Smoking_History}`, within($(" #History-and-Examinations ")), toRightOf("Smoking History")).exists())
 });
 
 step("Verify consultation notes in patient clinical dashboard", async function () {
     var consultationNote = gauge.dataStore.scenarioStore.get("consultationNotes")
-    await click(link(toLeftOf(text("OPD"), within($("#Visits")))))
+    await click(link(toLeftOf(text("OPD"), within($(" #Visits")))))
```

Comment:
unnecessary space before #

Reply:
To locate this element,I had to add space